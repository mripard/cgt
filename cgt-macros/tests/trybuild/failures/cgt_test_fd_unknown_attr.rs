#[cgt_macros::cgt_test_with_fd(unknown)]
fn test(_: std::os::fd::BorrowedFd<'_>) -> Result<(), TestResult> {
    Ok(())
}

fn main() {}