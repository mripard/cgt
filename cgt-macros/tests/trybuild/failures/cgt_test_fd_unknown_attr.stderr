error: supported fields are `master` and `capabilities`
 --> tests/trybuild/failures/cgt_test_fd_unknown_attr.rs:1:32
  |
1 | #[cgt_macros::cgt_test_with_fd(unknown)]
  |                                ^^^^^^^
